test/pytest/test_bridge.py

```diff
@@ -553,6 +555,21 @@ async def test_fsreplace1(transport: MockTransport, tmp_path: Path) -> None:
     await transport.assert_msg('', command='done', channel=ch)
     await transport.check_close(channel=ch)
 
+    # tag is "-" and file should not exist
+    newfile = tmp_path / 'new'
+    ch = await transport.check_open('fsreplace1', path=str(newfile), tag='-')
+    transport.send_data(ch, b'some stuff')
+    transport.send_done(ch)
+    await transport.assert_msg('', command='done', channel=ch)
+    await transport.check_close(channel=ch)
+    assert newfile.read_text() == 'some stuff'
+
+    # delete the now written newfile with tag='-' the file should now exist so this should fail
+    ch = await transport.check_open('fsreplace1', path=str(newfile), tag='-')
+    transport.send_done(ch)
+    await transport.assert_msg('', problem='change-conflict', channel=ch)
+    newfile.exists()
```

Comment:
That needs to be wrapped in `self.assertTrue()`, it's not an assertion by itself.